bundles/org.openhab.binding.resol/README.md

```diff
@@ -0,0 +1,196 @@
+ * System controller DeltaSol® SLT
+ * System controller DeltaSol® MX
+ * System controller DeltaSol® E
+ * DeltaSol Fresh®
+ * DeltaSol® Pool
+ * DeltaSol® Minipool
+ * DeltaSol® ES
+ * Frista
+ * DeltaTherm® HC
+ * DeltaTherm® FK
+ * Deltatherm® HT
+ * DeltaTherm® DH
+ * Sonnenkraft SKSC3
+ * Sonnenkraft STRG BX PLUS
+ * Sonnenkraft SKSC3+
+ * Sonnenkraft SKSC3HE
+ * Sonnenkraft SKSR 1/2/3
+ * Vitosolic 200
+ * COSMO Multi
+ * Drainback DeDietrich
+ * Diemasol C
+
+A more complete list can be found in the doc of the [resol-vbus-java library](http://danielwippermann.github.io/resol-vbus/vbus-packets.html).
+
+## Discovery
+
+Discovery is tested for VBus-LAN adapters DL2, DL3 and KM2 devices, it should also work for other devices providing a live data port.
+After a bridge is detected in the local network the password needs to be given and the things on the VBUS will popup in the inbox.
+
+## Binding Configuration
+
+The Resol binding doesn't need any form of configuration in files.
+
+## Bridge Configuration
+
+The bridge is the device connecting the Resol VBUS to the network, usually a VBus-LAN adapter or integrated in some of the solar controllers like DL3.
+For the connection from the Resol binding the bridge requires the configuration of
+
+| property             | type    | Required | description                                                |
+|----------------------|---------|----------|------------------------------------------------------------|
+| ipAddress            | String  | yes      | IP address or hostname of the VBUS adapter                 |
+| password             | String  | yes      | Password, defaults to 'vbus' for factory setting devices   |
+| port                 | Number  | no       | Port for the TCP connection, defaults to 7053              |
+| adapterSerial        | String  | no       | Serialnumber of the device (informative only)              |
+
+
+## Thing Configuration
+
+Depending on the solar/heating controller you have attached to your VBUS there will be a "controller" and several other things like heat quantity meters, heating circuit controls, etc.
+These do not require any configuration parameters and will pop up in your inbox after the bridge has received data from them.
+The name of the devices is usually the Resol name with spaced replaced by _ and a "-Controller" suffix like "DeltaSol_MX-Controller".
+For configuration in files you can enable the logging with at least INFO level for the resol binding and search the logs for "ThingHandler for (.*) not registered." to identify the names of the things you can add for your VBUS devices.
+
+
+## Channels
+
+The channels of a thing are determined automatically based on the received VBUS data and are highly dependent on the used device.
+Here is a list of the channels of a DeltaSol MX with a heat quantity meter (HQM) and an extension module EM.
+The channels supported for your device can be seen after autodiscovery or 
```

Comment:
good question :-) I have no clue how we should check it. Should we log the names in trace mode? - currently in trace the localized names - which are used as channel labels are printed. Maybe it would be better to change it output the channel name? Or should we just assume that any user is capable of looking the channels up in the web UI? - even those who still configure using text files?

I am tending to change the sentence to
**The channels supported for your device can be seen in the UI after data is received from the physical device.**

Reply:
I think logging in trace is ok for users who doesn't use an UI.